frontend-react/src/AppRouter.tsx

```diff
@@ -249,6 +256,14 @@ export const appRoutes: RouteObject[] = [
                     },
                 ],
             },
+            {
+                path: "/product/release-notes",
```

Comment:
Is release-notes staying under the product sub-directory or does it need to move to /about?

Reply:
@etanb is working on release notes and will fix in another PR